tasklist/qa/backup-restore-tests/src/test/java/io/camunda/tasklist/qa/backup/BackupRestoreTest.java

```diff
@@ -48,7 +50,8 @@
 import org.testcontainers.containers.output.Slf4jLogConsumer;
 
 @ExtendWith(SpringExtension.class)
-@ContextConfiguration(classes = {TestConfig.class})
+@ContextConfiguration(
+    classes = {TestConfig.class, UnifiedConfiguration.class, TasklistPropertiesOverride.class})
```

Comment:
:question: Why does this test need UnifiedConfiguration and TasklistPropertiesOverride? I don't see that it previously used TasklistProperties.

Reply:
TasklistPropertiesOverride declares the UnifiedConfiguration object. Even though it's not used, it's annotated with the config annotation and if I don't put it here, it fails to register it as a bean